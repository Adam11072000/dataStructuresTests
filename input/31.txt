init 2243
addImage 2945
getHighestScoredLabel 2945 1695
addImage 9604
addImage 7126
addImage 2945
resetLabelScore 2945 250 0
addImage -6
quit