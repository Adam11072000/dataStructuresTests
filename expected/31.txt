init done.
addImage: SUCCESS
getHighestScoredLabel: FAILURE
addImage: SUCCESS
addImage: SUCCESS
addImage: FAILURE
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
